apps/portals/src/configurations/elportal/synapseConfigs/data.ts

```diff
@@ -18,7 +18,6 @@ const data: SynapseConfig = {
     cavaticaConnectAccountURL: cavaticaConnectAccountURL,
     visibleColumnCount: 10,
     isRowSelectionVisible: true,
-    additionalFiltersLocalStorageKey: 'cohort-builder-files-perspective',
```

Comment:
This is a bug fix.  The V1 Files tab in the ELITE portal is not part of the cohort builder, and should not look at this special key to apply custom filters.